src/axom/inlet/Table.hpp

```diff
@@ -394,7 +432,7 @@ class Table : public Verifiable
    * \param [in] name Name of the dict
    * \param [in] description Description of the Field
    *
-   * \return Reference to the created Field
+   * \return Reference to the created dictionary
    *****************************************************************************
    */
   Verifiable& addBoolDict(const std::string& name,
```

Comment:
Suggestion/possibility: Would the more verbose `addBoolDictionary()` be more consistent with the rest of `inlet`?
E.g. the above functions are `get*Array()`, not `get*Arr()`

Reply:
My initial thinking on the name was that we'd use `dict` as Python does, but as we use `Array` instead of `list` already I don't have a problem changing it to `Dictionary`.